content/docs/istio/cr-istio.yaml

```diff
@@ -115,7 +115,6 @@ spec:
       statsd_address: localhost:9125
     ui: true
 
-  # See: https://github.com/bank-vaults/bank-vaults#example-external-vault-configuration for more details.
```

Comment:
Shall we replace this with https://bank-vaults.dev/docs/external-configuration/ ?

Reply:
Thanks for the tip, added it :)